nextflow_schema.json

```diff
@@ -433,6 +433,12 @@
                     "format": "file-path",
                     "exists": true
                 },
+                "variant_consequences_snv": {
+                    "type": "string",
+                    "description": "File containing list of SO terms listed in the order of severity from most severe to lease severe for annotating genomic SNVs.",
+                    "help_text": "For more information check https://grch37.ensembl.org/info/genome/variation/prediction/predicted_data.html",
```

Comment:
It says 38 somewhere else, is it to just take a look and understand how it works overall?

Reply:
Yeah, I just copied from raredisease, it's basically the same page but I can remove the `grch37`.